openapi/components/schemas/LegacyOneTimeSale.yaml

```
@@ -0,0 +1,215 @@
+  id:
+    type: string
+    description: ID of the order.
+    readOnly: true
+    maxLength: 50
+    example: ord_01GYJPRKHBD6ZYHH897QCJMBS4
+  orderType:
+    description: |-
+      Specifies the type of order.
+      An order may be a subscription or a one-time purchase.
+    type: string
+    x-basic: true
+    enum:
+      - one-time-order
+  customerId:
+    $ref: ./CustomerId.yaml
+  organizationId:
+    deprecated: true
+    readOnly: true
+    allOf:
+      - $ref: ./OrganizationId.yaml
+  status:
+    description: Status of the one-time order.
+    type: string
+    readOnly: true
+    enum:
+      - pending
+      - abandoned
+      - completed
+      - canceled
+  billingStatus:
+    description: |-
+      Billing status of the most recent invoice.
+      This value may help you to determine if you should change the service status of the service,
+      such as suspending the service.
+    type: string
+    readOnly: true
+    enum:
+      - draft
+      - unpaid
+      - past-due
+      - abandoned
+      - paid
+      - voided
+      - refunded
+      - disputed
+      - partially-refunded
+      - partially-paid
+  websiteId:
+    x-sortable: true
+    x-basic: true
+    allOf:
+      - $ref: ./WebsiteId.yaml
+  currency:
+    description: Currency of the order.
+    readOnly: true
+    $ref: ./CurrencyCode.yaml
+  initialInvoiceId:
+    description: ID of the initial invoice (`null` for one-time orders).
+    readOnly: true
+    type:
+      - 'string'
+      - 'null'
+    maxLength: 50
+    example: in_0YVF9605RKC62BP14NE2R7V2XT
+  recentInvoiceId:
+    description: |-
+      ID of the most recently issued invoice.
+      The invoice might not be `paid` yet.
+    readOnly: true
+    type:
+      - 'string'
+      - 'null'
+    maxLength: 50
+    example: in_0YVF9605RKC62BP14NE2R7V2XT
+  items:
+    description: Details of items in the order.
+    type: array
+    minItems: 1
+    items:
+      $ref: ./OrderItem.yaml
+  deliveryAddress:
+    description: Delivery address of the order.
+    oneOf:
+      - $ref: ./ContactObject.yaml
+      - type: 'null'
+  billingAddress:
+    description: Billing address of the order.
+    oneOf:
+      - $ref: ./ContactObject.yaml
+      - type: 'null'
+  activationTime:
+    description: Date and time when the order is activated.
+    x-sortable: true
+    type:
+      - 'string'
+      - 'null'
+    format: date-time
+    readOnly: true
+  voidTime:
+    description: Date and time when the order is voided.
+    type:
+      - 'string'
+      - 'null'
+    format: date-time
+    readOnly: true
+  abandonTime:
+    type:
+      - 'string'
+      - 'null'
+    description: |-
+      Date and time when the pending order is automatically abandoned.
+      If this value is not passed during order creation,
+      a [pending order TTL](https://www.rebilly.com/catalog/all/Organizations/PatchOrganization/#!t=request&path=settings/billing/pendingOrderTtl) setting is used to calculate the value.
```

suggestion: think we should use 'the' instead of 'a'

```suggestion
      the [pending order TTL](https://www.rebilly.com/catalog/all/Organizations/PatchOrganization/#!t=request&path=settings/billing/pendingOrderTtl) setting is used to calculate the value.
```